test/lib/usd/translators/UsdImportDisplacementTest/SimpleDisplacement.usda

```diff
@@ -0,0 +1,86 @@
+#usda 1.0
+(
+    defaultPrim = "pPlane16"
+    metersPerUnit = 0.01
+    upAxis = "Y"
+)
+
+def Mesh "pPlane16" (
```

Comment:
We have a much bigger issue here. This is exactly how Maya currently exports materials if you export a single mesh with a material. I did not hand-edit the file at all.

Reply:
Ah, I see. For our assets, we always organize everything under a single transform node when working in Maya so that when we export it to USD there's a definitive defaultPrim. This makes things a lot easier when you then author references to that USD.

I think this is where we figure out where to author the `Materials` scope:
https://github.com/Autodesk/maya-usd/blob/dc910a9da47299ceb916c8b0b0a12505b1ab958d/lib/mayaUsd/fileio/shading/shadingModeExporterContext.cpp#L336

Sounds like maybe after we've found a root prim path but **before** we append the materialsScopeName to it, we should check whether it's an `Xform`, and if it's not, just author the `Materials` scope under the `AbsoluteRootPath`?

Reply:
Yes. That would potentially solve an issue we have where "Looks" transforms are multiplying in Maya. You get one for each export-import cycle. I will copy this discussion to the internal bug we opened to track the breeding Looks and see if we can schedule it a bit earlier.